baybe/simulation/core.py

```diff
@@ -14,22 +15,20 @@
 from baybe.exceptions import NotEnoughPointsLeftError, NothingToSimulateError
 from baybe.simulation.lookup import _look_up_target_values
 from baybe.targets.enum import TargetMode
-from baybe.utils.basic import set_random_seed
 from baybe.utils.dataframe import add_parameter_noise
 from baybe.utils.numerical import closer_element, closest_element
+from baybe.utils.random import temporary_seed
 
-_DEFAULT_SEED = 1337
 
-
-def simulate_experiment(
+def simulate_experiment(  # noqa: DOC502
```

Comment:
Which error is this?

Reply:
This is an artifact that came from splitting the function (related to exceptions). Have removed it again. Thanks for spotting!